src/libman/Commands/BaseCommand.cs

```diff
@@ -216,5 +217,18 @@ protected void LogErrors(IEnumerable<IError> errors)
                 Logger.Log(string.Format("[{0}]: {1}", error.Code, error.Message), LogLevel.Error);
             }
         }
+
+        protected string GetUserInputWithDefault(string fieldName, string defaultFieldValue)
```

Comment:
I think we should update the disambiguating codepath to use this as well, even if it can't be hit today. If we do re-enable it in the future, it should work right.

Reply:
The disambiguation code path now handles this case and shows the below message: 
```
Cannot prompt for user input in this console.
Please specify the complete library id and the also the provider using '--provider' option.
```